nextjs-is-all-you-need.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Using Next.js to improve speed and efficiency at Xata'
+description: 'Discover how Xata utilizes Next.js to enhance speed and efficiency.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/nextjs-to-improve-speed-and-efficiency.jpg
+  alt: Buzz saying to Woody to infinity and beyond!
+author: Elizabet Oliveira
+date: 11-09-2023
+tags: ['design', 'nextjs', 'vercel']
+published: true
+slug: nextjs-to-improve-speed-and-efficiency
+---
+
+> The following post highlights some of the insights and experiences our Senior Product Designer, Elizabet Oliveira, shared at a recent talk at the Next.js Conf 20023
```

Comment:
We could consider embedding the talk here, early on in the blog.

Reply:
I already moved it. @joan-ing feel free to commit directly to the branch if you find more issues.